data/fwupd-agent.cron

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/usr/libexec/fwupdagent
```

Comment:
This should have `@libexecdir@`

Reply:
Since this thing will be running in cron, thoughts on maybe running as `nobody`?

Reply:
> This should have `@libexecdir@`

Fixed.

> thoughts on maybe running as `nobody`

What's the easiest way to do that? I did wonder if I was supposed to be using systemd for this rather than using cron.

Reply:
I like the idea of using a systemd timer much more.  There is way better confinement that you can do with systemd for a service that is talking out to the network..

Reply:
Agreed. I've switched to a timer now, which like you say also means we can ship everything in the main package.